app/Main.hs

```diff
@@ -199,9 +199,28 @@ updateImpl config@PackageConfig{ depends } = do
   echoT ("Updating " <> pack (show (length trans)) <> " packages...")
   forConcurrently_ trans . uncurry $ installOrUpdate (set config)
 
+pursCmd :: IO Text
+pursCmd = do
+  purs <- which "purs"
+  cmd <- which "purs.cmd"
+  exe <- which "purs.exe"
+  return $ fromMaybe "purs" $ msum [ constCmd "purs" purs
```

Comment:
Let's exit here instead of returning `purs`?

Reply:
Sure thing. Perhaps a message like "purs needs to be on your path"?

Reply:
Yes please!

Reply:
Maybe "The "purs" executable could not be found. Please make sure your PATH variable is set correctly"?